pkg/processor/build/builder.go

```diff
@@ -1283,3 +1299,56 @@ func (b *Builder) getSourceCodeFromFilePath() (string, error) {
 
 	return string(functionContents), nil
 }
+
+// addDependenciesCommand will add command to install dependencies if both DependenciesFile exists
+// and DependenciesCommand is not empty
+func (b *Builder) addDependenciesCommand(directives map[string][]functionconfig.Directive) error {
+	dependenciesFileName := b.runtime.GetDependenciesFileName()
+	if dependenciesFileName == "" {
+		b.logger.Debug("No runtime dependencies file name")
+		return nil
+	}
+
+	dependenciesFilePath := path.Join(b.stagingDir, "handler", dependenciesFileName)
+	if !common.FileExists(dependenciesFilePath) {
+		b.logger.Debug("Dependencies file not found")
+		return nil
+	}
+
+	b.logger.InfoWith("Found dependencies file", "path", dependenciesFilePath)
+	dependenciesCommand := b.options.FunctionConfig.Spec.Build.DependenciesCommand
+	if dependenciesCommand == "" {
+		dependenciesCommand = b.runtime.GetDependenciesCommand()
+	}
+
+	if dependenciesCommand == "" {
+		b.logger.Warn("No dependencies command")
+		// TODO: Should this be an error
+		return nil
```

Comment:
Yes, this should be an error. Prefer the process fails than succeeds and doesn't install requirements as expected, only to fail later